Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -456,13 +456,24 @@ The following are the settings for Storage Class Device Sets which can be config
 * `tuneDeviceClass`: For example, Ceph cannot detect AWS volumes as HDDs from the storage class "gp2", so you can improve Ceph performance by setting this to true.
 * `tuneFastDeviceClass`: For example, Ceph cannot detect Azure disks as SSDs from the storage class "managed-premium", so you can improve Ceph performance by setting this to true..
 * `volumeClaimTemplates`: A list of PVC templates to use for provisioning the underlying storage devices.
+    * `metadata.name`: If it is "metadata" and "wal", the devices are used to store metadata and WAL respectively. In both cases, the devices should be raw devices or LVM logical volumes.
     * `resources.requests.storage`: The desired capacity for the underlying storage devices.
-    * `storageClassName`: The StorageClass to provision PVCs from. Default would be to use the cluster-default StorageClass. This StorageClass should provide a raw block device, multipath device, or logical volume. Other types are not supported. If you want to use logical volume, please see [known issue of OSD on LV-backed PVC](../../Troubleshooting/ceph-common-issues.md#lvm-metadata-can-be-corrupted-with-osd-on-lv-backed-pvc)
+    * `storageClassName`: The StorageClass to provision PVCs from. Default would be to use the cluster-default StorageClass.
     * `volumeMode`: The volume mode to be set for the PVC. Which should be Block
     * `accessModes`: The access mode for the PVC to be bound by OSD.
 * `schedulerName`: Scheduler name for OSD pod placement. (Optional)
 * `encrypted`: whether to encrypt all the OSDs in a given storageClassDeviceSet
 
+Allowed configurations are as follows:
+
+| block device type | limitation                                                                          |
+|:------------------|:------------------------------------------------------------------------------------|
+| disk              |                                                                                     |
+| part              | `encrypted` should be `false`                                                       |
```

Comment:
Let's use "must" instead of "should" in these limitations, if you could also replace the others.
```suggestion
| part              | `encrypted` must be `false`                                                       |
```